src/theme/NavbarItem/LocaleDropdownNavbarItem/locale-dropdown-navbar-item.scss

```diff
@@ -69,12 +42,12 @@
     font-size: 16px;
     font-weight: normal;
     margin-top: 15px;
+    padding: 10px;
     color: var(--core-color-opacity-black-600, rgba(0, 0, 0, 0.72)) !important;
-    background-color: var(--ifm-navbar-background-color);
   }
 
   .dropdown__link--active {
-    color: var(--core-color-opacity-black-600, rgba(0, 0, 0, 0.72)) !important;
+    color: var(--colors-coral500) !important;
```

Comment:
`--colors-coral500` is this a quill token variable?

Reply:
Yes

Reply:
then this is incorrect. We only have these coral tokens: 

`--core-color-solid-coral-500` 
`--core-color-opacity-coral-500`